zerver/actions/streams.py

```diff
@@ -130,13 +92,6 @@ def do_deactivate_stream(stream: Stream, *, acting_user: Optional[UserProfile])
     # Get the affected user ids *before* we deactivate everybody.
     affected_user_ids = can_access_stream_user_ids(stream)
 
-    stream_subscribers = get_active_subscriptions_for_stream_id(
-        stream.id, include_deactivated_users=True
-    ).select_related("user_profile")
-    subscribed_users = [sub.user_profile for sub in stream_subscribers]
-    stream_subscribers.update(active=False)
```

Comment:
Any reason this was removed ? We would still want to update the activate field.
i see in `stream_subscription`, that we usually query the db using active=True, and there might be callers where we do not want to include archived stream.

Reply:
Ah I thought since we are not unsubscribing members we don't need to mark the subscription false. But ig you are right there might be callers where they don't want archived streams. I have reverted this change.

Reply:
Done.